src/client/common/utils/cacheUtils.ts

```diff
@@ -23,9 +23,9 @@ const resourceSpecificCacheStores = new Map<string, Map<string, CacheData>>();
  * Get a cache key specific to a resource (i.e. workspace)
  * This key will be used to cache interpreter related data, hence the Python Path
  *  used in a workspace will affect the cache key.
- * @param {String} keyPrefix
  * @param {Resource} resource
  * @param {VSCodeType} [vscode=require('vscode')]
+ * @param {IServiceContainer} serviceContainer
```

Comment:
```suggestion
 * @param serviceContainer
```

or add `{IServiceContainer}` to `getCacheStore`, up to you (I personally think we don't need to specify types, we already get them when hovering on the method name).